.all-contributorsrc

```diff
@@ -4597,6 +4597,7 @@
       ]
     }
   ],
```

Comment:
```suggestion
```
This list appears to be ended twice (and this one is too early 🙂)

Reply:
Ack, thanks Jim! I ran it through a JSON validator, and it caught the first error and not the second! 😮‍💨